bundle/compliance/policy/process/ensure_arguments_and_config.rego

```diff
@@ -0,0 +1,93 @@
+package compliance.policy.process.ensure_arguments_and_config
+
+import data.benchmark_data_adapter
+import data.compliance.lib.common as lib_common
+import data.compliance.policy.process.data_adapter
+
+process_args := data_adapter.process_args(benchmark_data_adapter.process_args_seperator)
+
+finding(rule_evaluation) = result {
+	data_adapter.is_kubelet
```

Comment:
Look at `package compliance.policy.process.ensure_arguments_contain_key_value`
It is a good example of an agnostic audit that has filters for different processes like:
```
apiserver_filter := data_adapter.is_kube_apiserver
controller_manager_filter := data_adapter.is_kube_controller_manager
scheduler_filter := data_adapter.is_kube_scheduler
etcd_filter := data_adapter.is_etcd
```
The audits that have the type inside the finding are not being currently shared by multiple processes, when we will need to share it then we can just follow suit.
I did not do it, because it would require us to add the filter in every rule that uses that audit (did not think that it was necessary if the audit is not shared by multiple processes).
If you still think that we should change it, I will.

Reply:
TY @jeniawhite,  I guess we can postpone it to the moment those audits will be used in different process' rules.